src/app/services/gameboards.tsx

```diff
@@ -151,10 +140,6 @@ export enum BoardCreators {
 
 export enum BoardSubjects {
     "all" = "All",
-    "physics" = "Physics",
-    "maths" = "Maths",
-    "chemistry" = "Chemistry",
-    "biology" = "Biology"
```

Comment:
is "all" a thing when the choice is a choice of one? And actually, where's computer science in this list?!

Reply:
Removed entirely